docs/examples/collect.py

```diff
@@ -166,6 +169,9 @@ def test_download_merge(self):
 parser.add_argument('--zoom', type=int, default=12,
                     help='Map zoom level given as integer. Defaults to 12.')
 
+parser.add_argument('--api-key', type=str, default=None,
```

Comment:
You can make this a required flag rather than an optional one, by skipping the “--” in the option name: https://docs.python.org/dev/library/argparse.html#name-or-flags.